\doxysection{Référence de la classe pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error}
\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}{}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}\index{pip.\_vendor.urllib3.exceptions.TimeoutError@{pip.\_vendor.urllib3.exceptions.TimeoutError}}


Graphe d\textquotesingle{}héritage de pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error\+:
% FIG 0


Graphe de collaboration de pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error\+:
% FIG 1


\doxysubsection{Description détaillée}
\begin{DoxyVerb}Raised when a socket timeout error occurs.

Catching this error will catch both :exc:`ReadTimeoutErrors
<ReadTimeoutError>` and :exc:`ConnectTimeoutErrors <ConnectTimeoutError>`.
\end{DoxyVerb}
 

La documentation de cette classe a été générée à partir du fichier suivant \+:\begin{DoxyCompactItemize}
\item 
env/lib/python3.\+12/site-\/packages/pip/\+\_\+vendor/urllib3/\mbox{\hyperlink{pip_2__vendor_2urllib3_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
